# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rbdoom3bfg/hotfix-vulkan-detail.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- neo/sys/DeviceManager_VK.cpp.vanilla	2024-11-21 16:59:00.190000006 +0100
+++ neo/sys/DeviceManager_VK.cpp	2024-11-21 16:58:26.600000001 +0100
@@ -1268,7 +1268,7 @@
 	}
 
 	// SRS - make static so ~DynamicLoader() does not prematurely unload vulkan dynamic lib
-	static const vk::DynamicLoader dl;
+	static const vk::detail::DynamicLoader dl;
 #endif
 	vkGetInstanceProcAddr = dl.getProcAddress<PFN_vkGetInstanceProcAddr>( "vkGetInstanceProcAddr" );
 	VULKAN_HPP_DEFAULT_DISPATCHER.init( vkGetInstanceProcAddr );
